Kickstarter-iOS/Views/Cells/DiscoveryEditorialCell.swift

```diff
@@ -0,0 +1,184 @@
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+      |> DiscoveryEditorialCell.lens.contentView.layoutMargins %~~ { _, cell in
+        cell.traitCollection.isRegularRegular
+          ? .init(top: Styles.grid(2), left: Styles.grid(30), bottom: 0, right: Styles.grid(30))
+          : .init(top: Styles.grid(2), left: Styles.grid(2), bottom: 0, right: Styles.grid(2))
+      }
+
+    _ = self.containerView
+      |> containerViewStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.editorialImageView
+      |> UIImageView.lens.contentMode .~ .scaleAspectFill
+
+    _ = self.editorialTitleLabel
+      |> editorialLabelStyle
+      |> \.font .~ UIFont.ksr_title2().bolded
+
+    _ = self.editorialSubtitleLabel
+      |> editorialLabelStyle
+      |> \.font .~ UIFont.ksr_callout()
+  }
+
+  func configureWith(value: DiscoveryEditorialCellValue) {
+    self.viewModel.inputs.configureWith(value)
+  }
+
+  // MARK: - Configuration
+
+  private func configureViews() {
+    _ = (self.containerView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    _ = (self.rootStackView, self.containerView)
+      |> ksr_addSubviewToParent()
+
+    _ = (self.editorialImageView, self.containerView)
+      |> ksr_addSubviewToParent()
+
+    _ = ([self.editorialTitleLabel, self.editorialSubtitleLabel], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    let tapGestureRecognizer = UITapGestureRecognizer(
+      target: self,
+      action: #selector(DiscoveryEditorialCell
+        .editorialCellTapped)
+    )
+
+    self.containerView.addGestureRecognizer(tapGestureRecognizer)
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.rootStackView.leftAnchor.constraint(equalTo: self.containerView.leftAnchor),
+      self.rootStackView.rightAnchor.constraint(equalTo: self.containerView.rightAnchor),
+      self.rootStackView.topAnchor.constraint(equalTo: self.containerView.topAnchor),
+      self.editorialImageView.leftAnchor.constraint(equalTo: self.containerView.leftAnchor),
+      self.editorialImageView.rightAnchor.constraint(equalTo: self.containerView.rightAnchor),
+      self.editorialImageView.bottomAnchor.constraint(equalTo: self.containerView.bottomAnchor),
+      self.editorialImageView.topAnchor.constraint(
+        equalTo: self.rootStackView.bottomAnchor,
+        constant: Styles.grid(1)
+      )
+    ])
+  }
+
+  // MARK: - Accessors
+
+  @objc private func editorialCellTapped() {
+    self.viewModel.inputs.editorialCellTapped()
+  }
+}
+
+// MARK: - Styles
+
+private let containerViewStyle: ViewStyle = { view in
+  view
+    |> roundedStyle(cornerRadius: Styles.grid(2))
+    |> \.backgroundColor .~ .ksr_trust_700
+    |> \.isAccessibilityElement .~ true
+    |> \.accessibilityTraits .~ [UIAccessibilityTraits.button]
```

Comment:
Open to other suggestions on how best to make this header accessible.

Reply:
I think `button` is closest to what we want.